nautobot/extras/templates/extras/job.html

```diff
@@ -19,7 +19,7 @@ <h1>{{ job }}</h1>
     {% if job.read_only %}
         <label class="label label-default">Read-only</label>
     {% endif %}
-    <p>{{ job.description }}</p>
+    <p>{{ job.description | render_markdown | placeholder }}</p>
```

Comment:
I don't think `placeholder` is needed here - I think it's probably preferable to have an empty line here rather than a "—" if no description is specified.